### PR TITLE
Add Native Flow Response reply type definition

### PR DESCRIPTION
**Pull request by pjvds**
This PR adds the type definition for an interactive nfp_reply message. This message is received when sharing an address, or completing an flow.

### CODE REVIEW

**Comment by Secreto31126** [comment]
Thanks for the PR! I didn't know this webhook notification existed. There are two nit details which will trigger a CI error, and one from the official documentation:

[Here](https://developers.facebook.com/docs/whatsapp/cloud-api/guides/send-messages/?locale=en#receive-notifications-for-address-submissions) it says the "interactive" object will also contain an "action" property, and the "nfm_reply" might contain a "name" and "body" strings.

If you don't mind, could you fix and add those extra types? If not, don't worry, I can keep working from this.

Thanks!

File: src/types.ts [hunk 1]
```diff
@@ -533,6 +533,12 @@ export type ServerInteractiveMessage = {
               };
               button_reply: never;
           };
```
Comment (Secreto31126) [inline src/types.ts:535]: There's an extra ; which should be in 541

**Review by Secreto31126 — CHANGES_REQUESTED** [review]

File: src/types.ts (hunk 1 above)
Comment (pjvds) [reply src/types.ts]: Great catch, thanks ✅

**Comment by Secreto31126** [comment]
Prefer using `unknown` over `any`, since it's type safer and doesn't make ESLint cry :)

Also, run `npm run prettier` to fix the nit issues.

**Comment by pjvds** [comment]
Got ya, will fix the issues and run Lu ting locally before proposing the final changes. Thanks!

**Comment by Secreto31126** [comment]
I added 2 more commits to the PR, super minor nit picks and replaced "address_response" with "address_message" based on the documentation.

Let me know if there's something else you would like to add, and if not I can merge it and release it tomorrow.

Thanks for contributing!

**Comment by pjvds** [comment]
Awesome, this all looks good now. I do have some other changes, but they are not related to this one. They have to deal with `post()` being non-blocking and serverless environments don't like those scenario's.

Anyhow... thanks for the great work!

**Pull request merged**